.circleci/config.yml

```diff
@@ -246,18 +352,24 @@ workflows:
                  - master
                  - dev
     jobs:
-      - build
-      - integration-presubmit-kind
+      - build_one_namespace
+      - build_isolated_namespaces
+      - integration-presubmit-one-namespace-kind
+      - integration-presubmit-isolated-namespaces-kind
       - integration-old-kind
       - install-minikube
       - install-demo
-      - noauth
+      - noauth_one_namespace
+      - noauth_isolated_namespaces
 
   all:
     jobs:
-      - build
-      - integration-presubmit-kind
+      - build_one_namespace
+      - build_isolated_namespaces
```

Comment:
Probably a single build is sufficient - it should build all combinations ( it does build demo and others )